examples/energy-management-app/energy-management-common/energy-evse/include/EnergyEvseDelegateImpl.h

```diff
@@ -36,6 +36,10 @@ namespace EnergyEvse {
 // chip::app::Clusters::EnergyEvse::TargetDayOfWeekBitmap)
 constexpr uint8_t kAllTargetDaysMask = 0x7f;
 
+// A sensible minimum limit for mains voltage (100V) to avoid accidental use
+// of 100V instead of 100000mV
```

Comment:
Did you mean to say `100mV`  in the incorrect example?

```suggestion
// of 100mV instead of 100000mV
```